web/client/libs/numeric-input/NumericInput.jsx

```diff
@@ -0,0 +1,1096 @@
+import React, { Component } from "react";
```

Comment:
Please indicate the source of this lib. 
```suggestion
/* 
 * Forked from https://github.com/vlad-ignatov/react-numeric-input
 */
import React, { Component } from "react";
```